docs/en/observability/apm/data-streams.asciidoc

```diff
@@ -34,6 +34,7 @@ Or, you might create namespaces that correspond to strategic business units with
 
 By type, the APM data streams are:
 
+// Does `traces-apm.rum-<namespace>` also apply to Android?
```

Comment:
Does `traces-apm.rum-<namespace>` also apply to Android in [APM data streams](https://observability-docs_3507.docs-preview.app.elstc.co/guide/en/observability/master/apm-data-streams.html#apm-data-streams-list)?

<img width="791" alt="Screenshot 2024-01-08 at 9 10 40 AM" src="https://github.com/elastic/observability-docs/assets/10479155/b06a2bc6-5602-45bf-8329-d625b569c21f">

Reply:
I believe so? @LikeTheSalad, can you confirm?

Reply:
I just checked with a local app and I noticed that Android's traces go into `traces-apm-default`, so it seems it's more in line with the other format for "Application traces" `traces-apm-<namespace>`.